packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -86,7 +86,63 @@ const getReadOnly = (
 }
 
 const ReflectionCard = (props: Props) => {
-  const {meeting, reflection, isClipped, openSpotlight, stackCount, showReactji, dataCy} = props
+  const {meetingRef, reflectionRef, isClipped, openSpotlight, stackCount, showReactji, dataCy} =
```

Comment:
Or I can use an alias for `relectionRef` during the destructuring of `Props`
Can you suggest any appropriate name for the alias